extensions/adobe/experience/intelligentServices/profile-journeyai-engagementscores.schema.json

```diff
@@ -0,0 +1,103 @@
+          "type": "object",
+          "isRequired": false,
+          "properties": {
+            "https://ns.adobe.com/experience/intelligentServices/journeyAI/engagementScores": {
+              "title": "engagementScores",
+              "description": "JourneyAI Engagement Scores provide likelihoods of a customer engaging with a newly delivered message.",
+              "type": "object",
+              "isRequired": false,
+              "properties": {
+                "https://ns.adobe.com/experience/intelligentServices/journeyAI/emailEngagementScores": {
+                  "title": "emailEngagementScores",
+                  "description": "Engagement scores for email channel messages",
+                  "type": "object",
+                  "isRequired": false,
+                  "properties": {
+                    "https://ns.adobe.com/experience/intelligentServices/journeyAI/emailClickEngagementScore": {
+                      "title": "emailClickEngagementScore",
+                      "description": "probability for a customer to click on links in an email",
+                      "type": "number",
+                      "isRequired": false,
+                      "meta:xdmType": "number"
+                    },
+                    "https://ns.adobe.com/experience/intelligentServices/journeyAI/emailOpenEnagementScore": {
+                      "title": "emailOpenEnagementScore",
+                      "description": "probability of a customer to open an email",
+                      "type": "number",
+                      "isRequired": false,
+                      "meta:xdmType": "number"
+                    },
+                    "https://ns.adobe.com/experience/intelligentServices/journeyAI/emailRetentionLevel": {
+                      "title": "emailRetentionLevel",
+                      "description": "qualitative categorization of customer's likelihood of staying subscribed to the email list",
+                      "type": "string",
+                      "isRequired": false,
+                      "meta:xdmType": "string"
+                    },
+                    "https://ns.adobe.com/experience/intelligentServices/journeyAI/emailRetentionPercentRank": {
+                      "title": "emailRetentionPercentRank",
+                      "description": "percentile rank of the likelihood of staying subscribed to the email list",
+                      "type": "number",
+                      "isRequired": false,
+                      "meta:xdmType": "number"
+                    },
+                    "https://ns.adobe.com/experience/intelligentServices/journeyAI/emailUnsubscriptionProbability": {
+                      "title": "emailUnsubscriptionProbability",
+                      "description": "the probability for recipient to unsubscribe from email channel given one email opened",
```

Comment:
Upper case the first letter. Ends with a "." period

Reply:
Done